packages/terra-clinical-result/translations/fi-FI.json

```diff
@@ -17,5 +17,6 @@
   "Terra.clinicalResult.interpretationPositive": "Positive",
   "Terra.clinicalResult.interpretationAbnormal": "Abnormal",
   "Terra.clinicalResult.interpretationHigh": "High",
-  "Terra.clinicalResult.interpretationLow": "Low"
+  "Terra.clinicalResult.interpretationLow": "Low",
+  "Terra.clinicalResult.bloodPressure": "Blood Pressure"
```

Comment:
That's weird that the Finnish translations are in English, though fixing it is beyond the scope of this PR.

Reply:
I didnt find any translations for Finnish, I have asked the same on the Jira - https://jira.cerner.com/browse/TRANS-14531?focusedCommentId=4755676&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4755676

Reply:
That's fine! We'll follow up on this on our end; We may no longer be supporting Finnish. Thanks btw!

Reply:
Thanks @sdadn